sdk/translation/azure-ai-translation-document/azure/ai/translation/document/models/_models.py

```diff
@@ -360,152 +508,207 @@ class StatusSummary(_model_base.Model):
     """Number of in progress. Required."""
     not_yet_started: int = rest_field(name="notYetStarted")
     """Count of not yet started. Required."""
-    cancelled: int = rest_field()
+    canceled: int = rest_field(name="cancelled")
```

Comment:
We don't rename body parameters. Why was this changed?

Reply:
This is response only. Changed for back compatibility with GA version.